packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/request.ts

```diff
@@ -120,12 +121,16 @@ export function textToText({
           sessionId: message.sessionId,
           messageId: message.messageId,
         });
+        this.eventSource = eventSource;
         for await (const event of toTextStream(eventSource, { timeout })) {
           if (event.type === 'message') {
             yield event.data;
           }
         }
       },
+      closeStream: function () {
+        this.eventSource?.close();
+      },
```

Comment:
Exposing EventSource is not a good idea. Better use a `signal` from `AbortController` instead.